java/geocoder/src/com/google/i18n/phonenumbers/PhoneNumberToTimeZonesMapper.java

```diff
@@ -172,7 +172,7 @@ public static String getUnknownTimeZone() {
    */
   private List<String> getTimeZonesForGeocodableNumber(PhoneNumber number) {
     List<String> timezones = prefixTimeZonesMap.lookupTimeZonesForNumber(number);
-    return Collections.unmodifiableList(timezones.isEmpty() ? UNKNOWN_TIME_ZONE_LIST
+    return Collections.unmodifiableList(timezones.size() == 0 ? UNKNOWN_TIME_ZONE_LIST
```

Comment:
This is a List, which was introduced in Java 1.2, at which time isEmpty was already there?
the isEmpty on string was indeed introduced in Java 1.6, so that we should avoid.

Was this an accidental "fix"? I notice PhoneNumberUtil uses isEmpty on lists, and that isn't listed here?